src/vs/workbench/contrib/testing/browser/testingExplorerView.ts

```diff
@@ -351,9 +352,19 @@ export class TestingExplorerView extends ViewPane {
 			}
 		}
 
-		// If there's only one group, don't add a heading for it in the dropdown.
-		if (participatingGroups === 1) {
-			profileActions.shift();
+		const menuActions: IAction[] = [];
+
+		const key = this.contextKeyService.createOverlay([]);
+		const menu = this.menuService.createMenu(MenuId.TestPythonConfigMenu, key);
+		const actions = menu.getActions({ renderShortTitle: true }).flatMap(entry => entry[1]);
+		if (actions.length > 0) {
+			// fill if there are any actions
```

Comment:
are these lines necessary? `getActions` just gets the actions that it'll fill in when you call `createAndFillIn` anyway